tensorrt_llm/_torch/modules/fused_moe/fused_moe_deepgemm.py

```
@@ -0,0 +1,347 @@
+        max_padded_tokens = (x.shape[0] + 128) // 128 * 128
+        permuted_data_tensor_padded = torch.empty(
+            (self.expert_size_per_partition, max_padded_tokens,
+             self.hidden_size),
+            dtype=self.dtype,
+            device='cuda')
+
+        masked_m, start_offsets = preprocess_after_permute(
+            expert_first_token_offset_tensor
+        )
+        m_max = (x.shape[0] + 127) // 128 * 128
+        expected_m = (token_selected_experts.numel() + self.expert_size_per_partition - 1) // self.expert_size_per_partition
```

Ruff (0.12.2)

Line too long (124 > 120)

(E501)